packages/lodestar/src/network/metadata.ts

```diff
@@ -48,53 +48,49 @@ export class MetadataController {
     this._metadata = opts.metadata || ssz.altair.Metadata.defaultValue();
   }
 
-  start(enr: ENR | undefined, currentFork: ForkName): void {
-    this.enr = enr;
-    if (this.enr) {
-      // updateEth2Field() MUST be called with clock epoch
-      this.updateEth2Field(this.chain.clock.currentEpoch);
-
-      this.enr.set(ENRKey.attnets, ssz.phase0.AttestationSubnets.serialize(this._metadata.attnets));
-      // Any fork after altair included
-
-      if (currentFork !== ForkName.phase0) {
-        // Only persist syncnets if altair fork is already activated. If currentFork is altair but head is phase0
-        // adding syncnets to the ENR is not a problem, we will just have a useless field for a few hours.
-        this.enr.set(ENRKey.syncnets, ssz.phase0.AttestationSubnets.serialize(this._metadata.syncnets));
-      }
-    }
+  /** Consumers that need the phase0.Metadata type can just ignore the .syncnets property */
+  get json(): altair.Metadata {
+    return this._metadata;
   }
-
   get seqNumber(): bigint {
     return this._metadata.seqNumber;
   }
-
+  get attnets(): BitArray {
+    return this._metadata.attnets;
+  }
+  set attnets(attnets: BitArray) {
```

Comment:
Please preserve whitespace in all changes of this file. After each method and at least 1 new line